samples/javascript_nodejs/17.multilingual-conversations/translator-middleware.js

```diff
@@ -57,7 +57,7 @@ class TranslatorMiddleware {
      */
     async translate(text, to) {
         // Check to make sure "en" is not translated to "in"
-        if (text.toLowerCase() === ENGLISH_LANGUAGE) {
+        if (text.toLowerCase() === ENGLISH_LANGUAGE || text.toLowerCase() === SPANISH_LANGUAGE) {
```

Comment:
Does the comment above this line need to be updated?

Reply:
I can update it, sure it was just for generality